src/v0/sources/shopify/transform.js

```diff
@@ -154,7 +154,7 @@ const processEvent = async (inputEvent, metricMetadata) => {
     default:
       if (!SUPPORTED_TRACK_EVENTS.includes(shopifyTopic)) {
         stats.increment('invalid_shopify_event', {
-          event: shopifyTopic,
+          writeKey: metricMetadata.writeKey,
```

Comment:
can't we use sourceId instead as writekey is little sensitive field ?

Reply:
The scope of change made was to only ensure we don't get prometheus errors, which means have all the labels mentioned in `prometheus.js` while firing stats from application.
In `prometheus.js` for this stat, we have `writeKey` as one of the labels. Hence the change was made
https://github.com/rudderlabs/rudder-transformer/blob/develop/src/util/prometheus.js#L526-L531

There are several instances in `shopify/transform.js` where we are using `writeKey` instead of `sourceId` & we also need to check if sourceId is being sent & use it. This is kind of a bigger change.